Docs/reference/content/reference/driver/crud/client_side_encryption.md

```diff
@@ -0,0 +1,193 @@
+
++++
+date = "2019-09-30T20:38:-04:00"
+title = "Client-Side Field Level Encryption"
+[menu.main]
+  parent = "Reference Reading and Writing"
+  identifier = "Client-Side Field Level Encryption"
+  weight = 40
+  pre = "<i class='fa fa-lock'></i>"
++++
+
+# Client-Side Field Level Encryption
+
+New in MongoDB 4.2, client-side field level encryption allows administrators and
+developers to encrypt specific data fields in addition to other MongoDB
+encryption features.
+
+With client-side field level encryption, developers can encrypt fields
+client-side without any server-side configuration or directives. Client-side
+field level encryption supports workloads where applications must guarantee that
+unauthorized parties, including server administrators, cannot read the encrypted
+data.
+
+{{% note class="important" %}} 
+Support for client-side field level encryption is in beta, and the beta only
+supports Windows. Backwards-breaking changes may be made before the
+final release (which will support MacOS and Linux).
+{{% /note %}}
+
+## mongocryptd configuration
+
+Client-side field level encryption requires the `mongocryptd` daemon / process
+to be running. A specific daemon / process uri URI be configured in the
```

Comment:
This doesn't seem right. FLE does *not* require that `mongocryptd` be running, because it will auto spawn it.

What IS required is that `mongocryptd.exe` be on the system path (or something similar on non-Windows platforms).

Reply:
Grammar error: ```A specific daemon / process uri URI be configured```

Missing the word "can"?

Reply:
Fixed!